variable-title.js

```diff
@@ -0,0 +1,30 @@
+window.onscroll = function () {
+    variableTitle()
+};
```

Comment:
za każdym razem jak idzie eventScroll, czyli baaardzo często, Twoje funkcje isBelow and isAbove od nowa siegają po węzły (document.querySelector('#'+selector)). A zapytania do drzewa DOM są kosztowne. Proponuje zapisać te węzły pod zmiennymi na początku pliku.
tzn:
var features = document.querySelector('#features'); 
itd. i te zmienne przekazywać jako parametr do metod isAbove i isBelow